stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -61,10 +57,23 @@ static inline void initializeBridgingFunctions() {
              nullptr);
 }
 
+id
+_swift_stdlib_createTaggedPointerString(const _swift_shims_UInt8 * _Nonnull bytes,
+                                        _swift_shims_CFIndex length) {
+  initializeBridgingFunctions();
+  if (_CFStringCreateTaggedPointerString != NULL) {
+    return _CFStringCreateTaggedPointerString(bytes, length);
+  }
+  return nil;
+}
+
 __swift_uint8_t
 _swift_stdlib_isNSString(id obj) {
   initializeBridgingFunctions();
-  return _CFGetTypeID((CFTypeRef)obj) == _CFStringTypeID ? 1 : 0;
+  if (_NSIsNSString != NULL) {
```

Comment:
This change is the big question mark I want benchmark coverage on. I think it should be similarly fast and cover some edge cases better.